### DISCUSSION TITLE
Oxygen CAN bus interface feature request

### DISCUSSION THREAD

**Discussion by flashtel** [iltis42/XCVario#204] [Ideas]
Hi,

The Oxygen sensor pictured is available now and I can assist with the PCB development and design for a CAN bus interface for it. I do not have the skills to modify the software for a display output and low alarm.

If this is something that the community would like to collaborate on I am keen, that is AUD so the sensor is around $180 euro.

![image](https://user-images.githubusercontent.com/17198499/184273125-2912a0f9-a1d0-447a-a780-e46f1b055e02.png)

Thanks
Paul

**Comment by flashtel** [comment]
I have made a prototype, it includes a dual channel current totaliser for lithium battery consumption.

The oxygen sensor I have a supply from china $25USD, they have to be cleaned for oxygen service. For those reading this do no buy Chinese sensors and hook them upto oxygen unless you know what your doing regards cleaning them. It can cause an explosion.

![signal-2022-11-24-230731_002](https://user-images.githubusercontent.com/17198499/204665800-82e52611-54d1-42f4-b70e-8f85db50c2b6.jpeg)

![signal-2022-11-24-230721_002](https://user-images.githubusercontent.com/17198499/204665901-7fe9285e-aec4-4f72-82c6-f2799c8aa4e9.jpeg)

![IMG_3625](https://user-images.githubusercontent.com/17198499/204669770-581d8b34-00c1-4d01-b9a6-d7a4bd0d5eea.jpg)

**Comment by iltis42** [comment]
Great idea, but better as a standalone gadget.
